dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -58,28 +94,34 @@ class WebhookIT extends BaseIT {
     @SystemStub
     public final SystemErr systemErr = new SystemErr();
 
-    private final String installationId = "1179416";
     private final String dockstoreTesting = "dockstore-testing";
-    private final String taggedToolRepo = "dockstore-testing/tagged-apptool";
-    private final String taggedToolRepoPath = "dockstore-testing/tagged-apptool/md5sum";
-    private final String workflowDockstoreYmlRepo = "dockstore-testing/workflow-dockstore-yml";
-    private final String testWorkflowsAndToolsRepo = "dockstore-testing/test-workflows-and-tools";
 
+    private FileDAO fileDAO;
+    private AppToolDAO appToolDAO;
 
     @BeforeEach
-    public void cleanDB() {
+    @Override
+    public void resetDBBetweenTests() {
```

Comment:
Overriding this method from BaseIT prevents the database from being reset twice before each test